sources/packages/api/src/services/student/student.service.ts

```diff
@@ -174,44 +186,102 @@ export class StudentService extends RecordDataModelService<Student> {
    * @returns Students pending SIN validation.
    */
   async getStudentsPendingSinValidation(): Promise<Student[]> {
-    return this.repo.find({ validSIN: null });
+    return this.repo
```

Comment:
I do not see a reason for this method to rely on the student table anymore.
Can we please have it moved to _SINValidationService_?

Reply:
This method requires most of the information from the student table, rather trying to find whether the student sin is pending or not. SO I believe it would he better to have it her in the student service, rather moving it to SINValidationService. If we start from SINValidationService, the relationship for the student data cannot be derived directly.

Reply:
About this: "trying to find whether the student sin is pending or not" we do the check of pending or not based on the sin_validation table, so the assumption is incorrect.

About this:  "If we start from SINValidationService, the relationship for the student data cannot be derived directly", yes it can be sin_validation->user->student

If you do not see a strong reason to change it you can keep it, but the justifications provided are not accurate.